package.json

```diff
@@ -19,7 +19,7 @@
     "watch:translations": "nodemon --watch apps/**/translations/src -x 'npm run hof-transpile'",
     "test": "NODE_ENV=test mocha",
     "test:ci": "npm run lint && npm run style && npm run test",
-    "test:acceptance": "codeceptjs run ./acceptance_tests --steps",
+    "test:acceptance": "codeceptjs run ./node_modules/hof-acceptance --steps",
```

Comment:
will that find it though? `codecept run` expects a path, AFAIK it doesn't auto-prepend node_modules so it would look in app/hof-acceptance which doesn't exist